docs/creating-multinode-controlplane-clusters.md

```diff
@@ -1,28 +1,46 @@
-# Creating Clusters With Multi-Node Control Plane  <!-- omit in toc -->
-Currently, clusterctl only supports creating clusters with a single instance control plane, but cluster-api-provider-aws has the capability to create clusters with multiple control plane nodes. This document outlines the steps for standing up a cluster with a multi-node control plane.
+# Creating clusters with a multi-node control plane  <!-- omit in toc -->
```

Comment:
Assuming all the line breaks are from some markdown formatting tool.

Reply:
It's a markdown linter, yeah